OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -118,7 +118,13 @@ public SlotDropDownOption(string title, string order, Func<bool> selected)
 				var flag = item.Get<ImageWidget>("FLAG");
 				flag.GetImageCollection = () => "flags";
 				flag.GetImageName = () => factionId;
-				item.GetTooltipText = () => faction.Description;
+
+				var displayName = faction.Description.SubstringBefore("\\n", StringComparison.Ordinal);
+				var displayDesc = faction.Description.SubstringAfter("\\n", StringComparison.Ordinal);
```

Comment:
displayName/Desc vs factionName/factionDescription  in SetupEditableFactionWidget